sites/partners/src/listings/PaperListingDetails/sections/DetailApplicationDates.tsx

```diff
@@ -16,14 +16,16 @@ const DetailApplicationDates = () => {
       <GridSection columns={3}>
         <GridCell>
           <ViewItem label={t("listings.applicationDeadline")}>
-            {listing.applicationDueDate &&
-              moment(new Date(listing?.applicationDueDate)).utc().format("MM/DD/YYYY")}
+            {listing.applicationDueDate
+              ? moment(new Date(listing?.applicationDueDate)).utc().format("MM/DD/YYYY")
+              : t("t.none")}
```

Comment:
Seems like this should somehow be able to be covered by `getDetailFieldString` – maybe a second optional argument?

Reply:
the typing here makes that hard to do cleanly, i made 2 new functions for this tho!